src/components/Match/matchColumns.jsx

```diff
@@ -541,11 +528,7 @@ export const inflictorsColumns = [{
     field: 'damage_inflictor',
     displayFn: (row, col, field) => (field ? Object.keys(field)
       .sort((a, b) => field[b] - field[a])
-      .map((k, i) => inflictorWithValue({
-        inflictor: k,
-        value: abbreviateNumber(field[k]),
-        key: i,
-      })) : ''),
+      .map(k => inflictorWithValue(k, abbreviateNumber(field[k]))) : ''),
```

Comment:
Regardless of the number of inputs, it's helpful to have the parameter name declared on invocation  so the developer doesn't have to look up the function signature.  (depending on your dev environment, your IDE might take care of this for you).

It's not a big deal to me, so if you like it better this way, we can keep it :)